pages/common/git-reflog.md

```diff
@@ -0,0 +1,11 @@
+# git reflog
+
+> Show when the refs were updated in local repository.
+
+- View reflog:
+
+`git reflog`
+
+- View N latest entries in reflog:
+
+`git reflog -n number_of_entries`
```

Comment:
Tokens should use the `{{token}}` format. Could you change this to `{{number_of_entries}}`?

Also, is `-n` a short hand for dry run?

Reply:
Also interesting, the man page seems to suggest that `-n` is for dry run (git/git@0ba014035aa693b546e51f242139b6641cf4718c) whereas in reality (and as per this example) seems to be the number of entries. 🤔

Reply:
Done!